services/operator_info.go

```diff
@@ -0,0 +1,134 @@
+package services
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Rocket-Rescue-Node/credentials"
+	"github.com/Rocket-Rescue-Node/credentials/pb"
+	"github.com/Rocket-Rescue-Node/rescue-api/models"
+	authz "github.com/Rocket-Rescue-Node/rescue-api/models/authorization"
+	"github.com/Rocket-Rescue-Node/rescue-api/util"
+	"go.uber.org/zap"
+)
+
+const (
+	// The maximum age for a operator info request to be considered valid.
+	operatorInfoRequestMaxAge = time.Duration(15) * time.Minute
+)
+
+type OperatorInfo struct {
+	CredentialEvents []int64 `json:"credentialEvents"`
+	QuotaSettings    *Quota  `json:"quotaSettings,omitempty"`
+}
+
+func CreateOperatorInfo(credentialEvents []int64, quotaSettings Quota) (*OperatorInfo, error) {
```

Comment:
function is exported, but only invoked in this package. Change to `createOperatorInfo`, or, better yet imo, delete the function and use `&OperatorInfo{CredentialEvents: credEvents}` style

Reply:
I like the inline approach. Much cleaner. Thanks for the suggestion!